spec/latest/core/core.jsonld

```diff
@@ -428,7 +434,7 @@
     {
       "@id": "hydra:IriTemplate",
       "@type": "hydra:Class",
-      "subClassOf": "hydra:Resource",
+      "subClassOf": ["hydra:Template", "hydra:Resource"],

```

Comment:
Do we need to mention hydra:Resource here? A hydra:Template is a hydra:Class is a hydra:Resource.

Reply:
I did not typed `hydra:Template` as any of the `hydra` types, thus to maintain backward compatibility, we have to point to `hydra:Resource` directly.